### ISSUE TITLE
hugepages support

### ISSUE THREAD

**Issue by liubin** [kata-containers/kata-containers#3342]
https://github.com/kata-containers/agent/pull/872

https://github.com/kata-containers/runtime/pull/3109

**Comment by lilongjiang1** [comment]
@liubin  thanks for your job. Do we need to mount /sys/fs/cgroup/hugetlb, and write hugepage limits to hugetlb cgroup for the container?

**Comment by liubin** [comment]
@lilongjiang1 Yes, the `resources` claimed by containers should be limited by cgroup in the guest OS.
